index.html

```diff
@@ -28,42 +38,52 @@ <h1 class="headline-title">Hey There.<br>I’m Martin</h1>
         <li class="button-head social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium.svg" alt="Medium icon"/></a></li>
       </ul>
     </section>
-    <section class="works-container">
+    <section class="works-container" id="Portfolio">
       <div class="works-title-container">
         <h2 class="works-title">
           My Recent Works
         </h2>
-      </div>
-      <div class="works-divider-container">
         <hr class="works-divider"/>
       </div>
-      <article class="work-container">
-        <div class="work-image-container">
-          <img src="./images/image2.png"/>
+      <article class="work-container work-main">
+        <div class="work-main-image-container">
+          <img class="work-main-image" src="./images/image2.png"/>
         </div>
-        <h3 class="work-title light">
-          Multi-Post Stories
-        </h3>
-        <p class="work-text light">
-          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="work-tag-group">
-          <li class="work-tag light">css</li>
-          <li class="work-tag light">html</li>
-          <li class="work-tag light">bootstrap</li>
-          <li class="work-tag light">Ruby</li>
-        </ul>
-        <div class="work-footer-container-light">
-          <a class="button">
+        <div class="work-main-content">
+          <h3 class="work-title light">
+            Multi-Post Stories
+          </h3>
+          <p class="work-text light">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="work-tag-group">
+            <li class="work-tag light">css</li>
+            <li class="work-tag light">html</li>
+            <li class="work-tag light">bootstrap</li>
+            <li class="work-tag light">Ruby</li>
+          </ul>
+          <div class="work-footer-container-light">
+            <a class="button">
+              See Project
+            </a>
+          </div>
+        </div>
+      </article>
+      <article class="work-container work-light sm-none">
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark fullwidth">
             See Project
           </a>
         </div>
       </article>
-      <article class="work-container bg-image">
+      <article class="work-container bg-image data-image">
```

Comment:
- [ ] Kindly make sure the spacing between both cards sticks to the one from the Figma design.

#### Figma Design 
![Screenshot from 2022-09-29 18-34-18](https://user-images.githubusercontent.com/83042742/193088523-19e11754-fede-4030-a943-155708c712f7.png)

#### Your Design 
![Screenshot from 2022-09-29 18-30-24](https://user-images.githubusercontent.com/83042742/193088767-c2519f58-e4af-4e2c-a430-6648d2c86189.png)